src/vtbackend/Functions.h

```diff
@@ -461,9 +465,13 @@ constexpr inline auto CaptureBufferCode = 314;
 
 // clang-format on
 
-inline auto const& functions() noexcept
+#if __cpp_consteval >= 201811L
+consteval auto functions() noexcept
+#else
+constexpr auto functions() noexcept
```

Comment:
I think it makes more sense to introduce a `CRISPY_CONSTEVAL` define that falls back to `constexpr` if `consteval` isn't supported.

Maybe this function should now be renamed to `allFunctions`, or simply `all`, because `functions` is now a little bit too synonymous.